taca/server_status/config.example.py

```diff
@@ -0,0 +1,23 @@
+import os
```

Comment:
hej @kate-v-stepanova is this file still needed? maybe we shoudl add a better example of configuraition.yaml file in the main TACA folder